backend/src/QA_integration.py

```diff
@@ -100,9 +101,9 @@ def QA_RAG(uri,userName,password,question,session_id):
         WITH d, apoc.text.join(collect(node.text),"\n----\n") as text, avg(score) as score
         RETURN text, score, {source: COALESCE(CASE WHEN d.url CONTAINS "None" THEN d.fileName ELSE d.url END, d.fileName)} as metadata
         """
-
+        embedding_model = os.getenv('EMBEDDING_MODEL')
         neo_db=Neo4jVector.from_existing_index(
-                embedding=OpenAIEmbeddings(),
+                embedding = VertexAIEmbeddings(model_name=embedding_model),
```

Comment:
shouldn't the embedding come from configuration now?

Reply:
already fixed in the newer branch